<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layoutCfgDialog</class>
 <widget class="QDialog" name="layoutCfgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>792</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>792</width>
    <height>430</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>窗口布局配置</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>283</y>
     <width>591</width>
     <height>125</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelChannel">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>通道</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxCameraType">
        <item>
         <property name="text">
          <string>拼接摄像机</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>高点摄像机</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>低点摄像机</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxChannel"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>布局列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxLayoutList"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnModify">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>修改</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>布局名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLayoutName">
        <property name="maxLength">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNewLayout">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>新建布局</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDoneNew">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>完成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelNew">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>168</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
